util-core/src/test/scala/com/twitter/util/CancellableTest.scala

```diff
@@ -3,6 +3,39 @@ package com.twitter.util
 import org.scalatest.WordSpec
 
 class CancellableTest extends WordSpec {
+  "Cancellable" should {
+    "test variable isCancelled" in {
+      assert(!Cancellable.nil.isCancelled)
+    }
+    "test method cancel" in {
```

Comment:
nit: please ensure there is line spacing between the tests (see [FlaggableTest](https://github.com/twitter/util/blob/develop/util-app/src/test/scala/com/twitter/app/FlaggableTest.scala) as an example)